Lib/test/test_syntax.py

```diff
@@ -2175,6 +2175,36 @@ def f(x: *b)
    ...
    SyntaxError: Type parameter list cannot be empty
 
+   >>> type A[1] = ...
+   Traceback (most recent call last):
+   ...
+   SyntaxError: expected a type parameter definition, found literal
+
+   >>> type A[*()] = ...
+   Traceback (most recent call last):
+   ...
+   SyntaxError: expected a type parameter definition, found tuple
+
+   >>> type A[**{}] = ...
+   Traceback (most recent call last):
+   ...
+   SyntaxError: expected a type parameter definition, found dict literal
+
+   >>> def some["abc"](): ...
+   Traceback (most recent call last):
+   ...
+   SyntaxError: expected a type parameter definition, found literal
+
+   >>> class Some[*...]: ...
```

Comment:
Should we increase the coverage so that we also check all possible pairs of inputs, such as `class Some[**{}]` or `def some[*()]`?

Reply:
Not really ;)